app/src/main/res/raw/testing_credentials_develop.json

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "server_url": "https://play.dhis2.org/android-previous2",
+    "user_name": "",
+    "user_pass": "",
+    "server_version": "2.29"
+  },
+  {
+    "server_url": "https://play.dhis2.org/android-previous1",
+    "user_name": "android",
+    "user_pass": "Android123",
+    "server_version": "2.30"
+  },
+  {
+    "server_url": "https://play.dhis2.org/android-current",
+    "user_name": "android",
+    "user_pass": "Android123",
+    "server_version": "2.31.3"
+  },
+  {
+    "server_url": "https://latest.malaria-uio.org",
+    "user_name": "josemp10",
+    "user_pass": "Dsme2018..",
+    "server_version": "2.32"
+  },
+  {
+    "server_url": "https://staging.hmisocba.msf.es",
+    "user_name": "tablet",
+    "user_pass": "Test@123",
+    "server_version": "2.32"
+  },
+  {
+    "server_url": "https://sandbox.psi-mis.org",
+    "user_name": "zwmcnkulumane",
+    "user_pass": "PSIzim2016@",
+    "server_version": "2.32"
+  },
+  {
+    "server_url": "https://training.dhis.dhmis.org/za-idsr",
+    "user_name": "Android_Tester",
+    "user_pass": "Android1234$",
+    "server_version": "2.32"
+  },
+  {
+    "server_url": "http://android2.dhis2.org:8080",
+    "user_name": "android",
+    "user_pass": "Android123",
+    "server_version": "2.32"
+  },
+  {
+    "server_url": "https://training.dhis.dhmis.org/za-idsr",
+    "user_name": "Android_Tester",
+    "user_pass": "Android1234$",
+    "server_version": "2.32"
+  },
+  {
+    "server_url": "https://moph-dw.gov.af",
+    "user_name": "idsr.test",
+    "user_pass": "Idsr@123",
+    "server_version": "2.32"
+  }
+]
```

Comment:
This file should not be included in the repo. Please remove it!

😰

Reply:
It's true. Thank you :sweat_smile: